test/e2e/network/ingressscale/localrun/main.go

```diff
@@ -0,0 +1,177 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strconv"
+
+	"github.com/golang/glog"
+
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	gcecloud "k8s.io/kubernetes/pkg/cloudprovider/providers/gce"
+
+	"k8s.io/kubernetes/test/e2e/framework"
+	scale "k8s.io/kubernetes/test/e2e/network/ingressscale"
```

Comment:
why don't we just rename this to "scale" package and have it also contain other networking scale related utils as well (later)

Reply:
Sounds good, renamed to "scale".